doc/installing.rst

```diff
@@ -9,11 +9,9 @@ Installation
 
 .. warning::
 
-    **Apple Silicon users** have had issues in the past installing PypeIt using
-    pip; however, we do not think this is an issue any longer.  If you have
-    trouble installing PypeIt on your Apple machine following the pip
-    installation instructions, first try following the `conda`_ instructions
-    instead.  If that also fails, please `Submit an issue`_ and/or reach out to
+    **Apple Silicon users** who are having issues installing PypeIt may need
+    to set up an environment configured for x86-64.  See :ref:`m1_macs` for detailed
```

Comment:
Is this `m1_macs` referenced to something? I can't seem to find where it is linked to.

Reply:
Down a little -- new line 170, the section header "User Installation on Apple Silicon-based Macs"